Documentation/dev-guide.md

```diff
@@ -79,3 +79,74 @@ DOCKER_LINKS=dex_postgres:postgres DOCKER_ENV=DEX_TEST_DSN ./go-docker ./test-fu
 # Remove the container after the tests are run.
 docker rm -f dex_postgres
 ```
+
+## Vendoring dependencies
+
+dex uses [godep](https://github.com/tools/godep) for vendoring all external dependencies. This document goes through the steps of adding and updating dex dependencies.
```

Comment:
Dex uses [godep](https://github.com/tools/godep) to vendor external dependencies. This section details how to add and update those dependencies.